lib/Tool/Serialize.php

```diff
@@ -102,7 +102,9 @@ protected static function loopFilterCycles($element)
             $propCollection = get_object_vars($clone);
 
             foreach ($propCollection as $name => $propValue) {
-                $clone->$name = self::loopFilterCycles($propValue);
+                if(is_array($propValue) || (is_object($propValue))){
+                    $clone->$name = self::loopFilterCycles($propValue);
+                }
```

Comment:
It doesn't look the right solution, because looking at `loopFilterCycles` could and should return the same `$propValue` when it's not object nor array, and in your case, it may have worked because it skipped the second-level recursion but looking at the error `'Cannot access property starting with "\0"'` that you have mentioned, it looks quite similar to this https://github.com/deliciousbrains/better-search-replace/issues/71, therefore the suggested solution there could also helpful here

```suggestion
                $name = trim($name);
                $clone->$name = self::loopFilterCycles($propValue);
```


Would you be able to retry with this approach?

Reply:
@kingjia90 your solution seems to work too 👍